Modules/Directory.Build.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+    <ItemGroup Condition="'$(Configuration)' == 'Release' And !$(MSBuildProjectFile.Contains('Tests.csproj'))">
```

Comment:
I could imagine that this condition causes the increased build times. 
For every single file in the solution, the compiler has to perform this check. I don't know how expensive such a check is during compilation, but it might be a possibility.

Reply:
To verify this, you could temporarily remove the check. While this will blow up the size of the dll, it should decrease the build time if my assumption is correct.
Btw: this also explains why the build time of the whole solution increases so drastically while the one for only the Consumer API only increases a little bit (the whole solution obviously has more files to check)

Reply:
Just did this check, build times are almost the exact same, so I'd say it's safe to assume this is not the culprit.

Reply:
With "this check" you mean "removing the entire condition"?

Reply:
Yes, removed the `Directory.Build.targets` file.